apis/v1alpha1/gateway_types.go

```diff
@@ -292,6 +294,26 @@ type Listener struct {
 	// are compatible with the application protocol specified in
 	// the Protocol field.
 	//
+	// Although a client request may technically match multiple route rules,
+	// only one may ultimately receive the request. Matching precedence MUST be
```

Comment:
`only one may ultimately receive the request`, clarify `one` as a route or a rule.

Reply:
Updated, thanks!